Dockerfile

```diff
@@ -51,37 +51,13 @@ RUN \
 
 ADD . /usr/local/src/scorum
 
-RUN \
-    cd /usr/local/src/scorum && \
-    mkdir build && \
-    cd build && \
-    cmake \
-        -DCMAKE_BUILD_TYPE=Release \
-        -DBUILD_SCORUM_TESTNET=ON \
-        -DLOW_MEMORY_NODE=OFF \
-        -DCLEAR_VOTES=ON \
-        -DSKIP_BY_TX_ID=ON \
-        .. && \
-    make -j$(nproc) && \
-    ./libraries/chainbase/test/chainbase_test && \
-    ./tests/utests/utests && \
-    ./tests/chain_tests/chain_tests && \
-    ./tests/wallet_tests/wallet_tests && \
-    ./programs/util/test_fixed_string && \
-    cd /usr/local/src/scorum && \
-    doxygen && \
-    programs/build_helpers/check_reflect.py && \
-    programs/build_helpers/get_config_check.sh && \
```

Comment:
move all these checks:

    doxygen && \
    programs/build_helpers/check_reflect.py && \
    programs/build_helpers/get_config_check.sh && \

 in to uncommented section. Check result: 

sudo docker build . -t test